todo_app/data/db_items.py

```diff
@@ -0,0 +1,87 @@
+from flask import session
+import pymongo
+import os
+from  todo_app.data.Item import Item
+from datetime import datetime
+
+
+max_id = 0
+
+def getDb():
+    client = pymongo.MongoClient(os.getenv('AZ_MONGODB_CONNECTION'))
+    db = client[os.getenv('AZ_MONGODB_DB_NAME')]
+    return db
+
+def get_items():
+    """
+    Fetches items from the connected DB.
+
+    Returns:
+        list: The list of saved items.
+    """     
+    db = getDb()
+
+    mdb_items_col = db.items
+    mdb_items = mdb_items_col.find({})
+    
+    cards = []
+
+    global max_id
+
+    for it in mdb_items:
+        if int(it['id_short']) > max_id:
+            max_id = int(it['id_short'])
+        cards.append(Item(it['id_short'], it['name'], it['status']))
+
+    return cards
+
+def add_item(title):
+    """
+    Adds a new item with the specified title to the connected database.
+
+    Args:
+        title: The title of the item.
+
+    Returns:
+        item: The saved item.
+    """
+    db = getDb()
+
+    mdb_items_col = db.items
+
+    global max_id
+
+    card = {
+        "id_short": max_id + 1,
+        "name": title,
+        "status": "To Do"
+    }
+
+    post_id = mdb_items_col.insert_one(card).inserted_id
+
+    return post_id
+
+def move_card_to_new_list(card_short_id, current_list_name, new_list_name):
```

Comment:
It looks like this function ignores the `current_list_name` variable, so it might make sense to just remove that argument